.dockerignore

```diff
@@ -1,8 +1,6 @@
 artifacts
 data
+docs
 
 .adr
-.yarn
-docs
-node_modules
-packages
```

Comment:
Are we sure we no longer want to dockerignore these directories?

Reply:
Good catch. I removed `node_modules`. Explanation on others below:

* `.yarn`: needed in order to build the gui
* `docs`: still in .dockerignore file just got moved to another line
* `node_modules`: not needed. Removed
* `packages`: needed in order to build the gui

Reply:
Great. I wonder if that was responsible for the 19MB increase in image size (the GUI itself is much smaller than that so I was a little surprised).